### PR TITLE
feat(context-helper): add entities and intent generation function

### PR DESCRIPTION
**Pull request by Palashx**
This function extracts entities and intents from training data and the stories, which will be used by the Context API to pass values to the individual components

Test cases:
1. No stories present
2. common_examples empty in the training data
3. only one model present with multiple combination of entities
4. Multiple models with combination of multiple entities [no entities, same entity present at multiple places at different places, different entities present at different places]

### CODE REVIEW

File: botfront/imports/api/story/stories.methods.js [hunk 1]
```diff
@@ -18,4 +18,14 @@ Meteor.methods({
...
         return Stories.remove(story);
     },
+
+    'stories.getStories'(projectId) {
+        check(projectId, String);
+        return Stories.find({ projectId }).fetch();
+    },
 });
+
+export const getStories = async (projectId) => {
```
Comment (MatthieuJnon) [inline botfront/imports/api/story/stories.methods.js]: why did you create this method if it does the exact same thing that the one above ??

File: botfront/imports/api/project/project.methods.js [hunk 2]
```diff
@@ -14,8 +14,62 @@ import { Deployments } from '../deployment/deployment.collection';
...
+        const models = NLUModels.find(
+            { _id: { $in: nluModelIds } },
+            { fields: { training_data: 1 } },
+        ).fetch();
+
+        // get commmon examples from the training data
+        let commonExamples = models.map(model => model.training_data.common_examples);
+
+        if (commonExamples.length !== 0) {
+            commonExamples = commonExamples.reduce((acc, x) => acc.concat(x));
```
Comment (MatthieuJnon) [inline botfront/imports/api/project/project.methods.js]: please use variables with more explicit names

File: botfront/imports/api/story/stories.methods.js (hunk 1 above)
Comment (Palashx) [reply botfront/imports/api/story/stories.methods.js]: I wrapped the meteor call, read online its a good practice to wrap meteor calls and call the wrapper function from other places. But now I also don't see much point doing so unless we are going to call that method several times.

File: botfront/imports/api/project/project.methods.js (hunk 2 above)
Comment (Palashx) [reply botfront/imports/api/project/project.methods.js]: Will change them

**Pull request merged**